Cargo.toml

```
@@ -32,3 +32,4 @@ derive_builder = "0.20.0"
 paste = "1.0.7"
 crc32c = "0.6.4"
 anyhow = "1.0.80"
+docker-compose-runner = "0.3.1"
```

We shouldn't include this in our regular deps, right?

```suggestion

[dev-dependencies]
docker-compose-runner = "0.3.1"
```

---

whoops, thanks for catching this!